models/demos/llama3/lt

```diff
@@ -262,11 +289,18 @@ def main(stdscr):
 
     max_y, max_x = stdscr.getmaxyx()
 
+    host_device = get_host_device()
+
     # Input fields positions (reordered)
     input_fields = [
         {"label": "Command [demo]", "value": "", "x": 0, "y": 0},
         {"label": "Model (1b, 3b, 8b, 11b, 70b) [all]", "value": "", "x": 0, "y": 1},
-        {"label": "Device (n150, n300, t3k) [all]", "value": "", "x": 0, "y": 2},
+        {
+            "label": "Device (n150, n300, t3k, tg) [all]" if host_device == "TG" else "Device (n150, n300, t3k) [all]",
```

Comment:
I don't think we need to differentiate if TG here. If you're running tests on an unsupported machine, the tests should not be called. 

@yieldthought  Thoughts?